components/StyledInputAmount.js

```diff
@@ -43,30 +45,68 @@ const getError = (curVal, minAmount, required) => {
   return Boolean((required && isNil(curVal)) || (minAmount && curVal < minAmount));
 };
 
+const generateCurrencyOptions = availableCurrencies => {
+  return Object.keys(availableCurrencies).map(key => {
+    const availableCurrency = availableCurrencies[key];
+    return {
+      value: key,
+      label: (
+        <Flex fontSize="14px" lineHeight="20px" fontWeight="500" title={availableCurrency.fullName}>
```

Comment:
This `title` will not work, `FormattedMessage` returns a react component, not a string. So currently `title="object [Object]"`